doc/stanzas/library.rst

```diff
@@ -106,9 +106,10 @@ order to declare a multi-directory library, you need to use the
 
 - ``(modes <modes>)`` is for modes which should be built by default. The most
   common use for this feature is to disable native compilation when writing
-  libraries for the OCaml toplevel. The following modes are available: ``byte``,
-  ``native`` and ``best``. ``best`` is ``native`` or ``byte`` when native
-  compilation isn't available.
+  libraries for the OCaml toplevel. It can also be used to define
+  libraries that should be compiled with Melange (see :ref:`melange_main`). The
```

Comment:
If the document is linked somewhere, I think it should be put in the toctree. you can start the document with a `.. warning::` directive to explain what kind of stability to expect.

Reply:
I removed this modification in the latest version of the docs.